times_reader/main.py

```diff
@@ -110,25 +110,26 @@ def convert_xl_to_times(
 
     transform_list = [
         lambda tables: dump_tables(tables, os.path.join(output_dir, "raw_tables.txt")),
+        transforms.generate_dummy_processes,
```

Comment:
We need to consider replacing the list with the dependency graph at some point but not now